### PR TITLE
Alert Spacing

### PR DESCRIPTION
**Pull request by MagdalenaHenke**
[Clubhouse](https://app.clubhouse.io/blocks/story/34418/alerts-fixes)
[Zeplin](https://app.zeplin.io/project/5adf8fe14c3924b93d974cba/screen/5b43ae0ddd03f30b025bd725)

1. Aligning text and icon
2. Adding extra space under alert title
3. Adding extra space under alert text

Also made all of the close icons to have a hover color. 

Before:
Screen Shot 2019-06-17 at 1 14 48 PM

After: 
Screen Shot 2019-06-17 at 3 23 45 PM

### CODE REVIEW

File: packages/blocks-base/styles/alerts.styl [hunk 1]
```diff
@@ -16,32 +16,32 @@
 .blx-alert-icon-container {
   position: relative;
   flex: 0 0 auto;
-  margin: 12px 16px;
+  margin: 16px;
   width: 32px;
   height: 32px;
   border-radius: 50%;
   background-color: $light-01;
-
```
Comment (MagdalenaHenke) [inline packages/blocks-base/styles/alerts.styl]: Sorry my sublime auto deletes extra white space.

**Comment by MagdalenaHenke** [comment]
This one is waiting for some Craig comments to get back to me.

**Comment by MagdalenaHenke** [comment]
@emma-floored @Jinsoo94 now this is ready for looks

**Comment by Jinsoo94** [comment]
looks good to me 👍

**Comment by emma-floored** [comment]
👍

**Pull request merged**